feeds/src/state/ducks/aggregator/fluxOperations.ts

```diff
@@ -0,0 +1,268 @@
+import { FeedConfig } from 'config'
+import { ethers } from 'ethers'
+import _ from 'lodash'
+import FluxAggregatorAbi from '../../../contracts/FluxAggregatorAbi.json'
+import FluxAggregatorContract from '../../../contracts/FluxAggregatorContract'
+import * as actions from './actions'
+
+export default class FluxOperations {
+  static contractInstance: any
+
+  static fetchOracleList() {
+    return async (dispatch: any, getState: any) => {
```

Comment:
Once the return type is typed as `ThunkAction<...>` we can remove the `any` here and it'll be properly typed

Reply:
@hellobart lets add the `ThunkAction<...>` return type here